payments/api_test.py

```diff
@@ -765,7 +695,7 @@ def test_get_auto_apply_discount_for_basket_auto_discount_exists_for_user():
 def test_get_auto_apply_discount_for_basket_multiple_auto_discount_exists_for_user_product_system():
     """
     Test that get_auto_apply_discount_for_basket returns multiple auto discount
-    when they exist for the basket's - basket item - product, basket's user, and basket's integrated system.
+    when they exist for the basket"s - basket item - product, basket"s user, and basket"s integrated system.
```

Comment:
Nit: not sure why these got changed to double-quotes here and in the 3 tests preceding. (No need for re-review for this.)